src/test/java/com/carservice/manager/service/VehicleFileServiceTest.java

```diff
@@ -0,0 +1,108 @@
+package com.carservice.manager.service;
+
+import com.carservice.manager.model.VehicleModel;
+import com.carservice.manager.repository.VehicleFileRepository;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+
+@SpringBootTest
```

Comment:
I think this annotation can be removed and we can unit test VehicleFileService class.